src/org/opendatakit/briefcase/ui/SettingsPanel.java

```diff
@@ -20,6 +20,7 @@
 import javax.swing.event.ChangeListener;
 
 import org.apache.http.HttpHost;
+import org.opendatakit.briefcase.model.BriefcaseAnalytics;
```

Comment:
Thanks for pointing that out :+1:, I ran the tests before I removed the instantiation of `BriefcaseAnalytics` with the expectation that such a small change would be of no consequence. I suppose unnecessary imports increase the memory footprint and increase the risk of a naming conflict.